CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/plugins/io/SaveGraphPlugin.java

```diff
@@ -42,30 +46,49 @@
 public class SaveGraphPlugin extends SimplePlugin {
 
     public static final String GRAPH_PARAMETER = PluginParameter.buildId(SaveGraphPlugin.class, "graphId");
+    public static final String FILE_PATH_PARAMETER = PluginParameter.buildId(SaveGraphPlugin.class, System.getProperty("user.home"));
 
     @Override
     public PluginParameters createParameters() {
         final PluginParameters parameters = new PluginParameters();
 
         final PluginParameter<StringParameterValue> graphIdParameter = StringParameterType.build(GRAPH_PARAMETER);
+        final PluginParameter<StringParameterValue> filePathParameter = StringParameterType.build(FILE_PATH_PARAMETER);
         graphIdParameter.setName("graphId");
         graphIdParameter.setDescription("The Id of the graph");
         parameters.addParameter(graphIdParameter);
+        filePathParameter.setName("SaveGraphPluginFilePath");
```

Comment:
More a personal preference but maybe naming it "saveFilePath"? Up to you there

Reply:
And separate it from `graphIdParameter` - they should be in distinct blocks for readablitily.

Reply:
It also doesn't look like you're adding it to the parameters object?